src/Eto.Mac/build/BundleDotNetCore.targets

```diff
@@ -24,12 +24,12 @@
   <Target Name="MacBuildAppBundle" AfterTargets="AfterBuild" DependsOnTargets="$(MacBuildAppBundleDependsOnTargets);_MacBuildFrameworkAppBundle;_MacBuildUnifiedAppBundle" Condition="$(MacAutoPublishBundle) == 'True' AND $(MacIsBuildingBundle) != 'True'" Outputs="$(OutputAppPath)">
   </Target>
   
-  <Target Name="_MacBuildUnifiedAppBundle" Condition="@(MacBundleRuntimeIdentifiers->Count()) > 1">
+  <Target Name="_MacBuildUnifiedAppBundle" Condition="@(MacBundleRuntimeIdentifiers->Count()) > 1 AND $(PublishSingleFile) == True">
```

Comment:
This is implied by this comment:

https://github.com/picoe/Eto/blob/f5d7804858d3df2e0213912c89e4d3dc20d09ae9/src/Eto.Mac/build/BundleDotNetCore.targets#L148-L150